src/RhythmRunner.cpp

```diff
@@ -132,7 +134,7 @@ int main(int argc, char** argv) {
           }
           game_state = std::make_shared<GameState>(
               level_generator->generateLevel(), std::make_shared<GameCamera>(),
-              std::make_shared<Player>(), std::make_shared<Sky>());
+              std::make_shared<Player>(), std::make_shared<Sky>(), window);
           // Only Video texture we have right now
           std::shared_ptr<VideoTexture> vid = std::make_shared<VideoTexture>(
```

Comment:
hmm I get a segfault when I take it out I think the sky is being treated as a video texture